cmd-gen.sh

```diff
@@ -0,0 +1,15 @@
+git clone https://github.com/TheThingsNetwork/lorawan-stack.git lorawan-stack
+
+cd lorawan-stack
+
+make init
```

Comment:
This will take a long time, and I don't think it's actually necessary to just generate some docs